kf_lib_data_ingest/common/datafile_readers.py

```diff
@@ -2,15 +2,24 @@
 Contains file readers (file -> dataframe) for file types that benefit from
 extra help.
 """
+import io
 import os
 
 import pandas
 import xlrd
 from pandas.api.types import is_file_like
 
 
+def __ascii_wrap(potentially_binary_buffer):
```

Comment:
So why do you need this? I thought `pandas.read_csv` can read binary streams already

Reply:
It can't with engine='python'.

Reply:
See also https://github.com/pandas-dev/pandas/issues/23779

Reply:
I see 👍